libs/s25main/mapGenerator/RandomConfig.cpp

```diff
@@ -22,11 +22,16 @@
 #include "gameData/WorldDescription.h"
 #include <boost/bind.hpp>
 #include <boost/random/uniform_real_distribution.hpp>
+
+#ifdef _WIN32
```

Comment:
What? Really? AFAIK pragmas were invented to implement compiler specific stuff and has to be ignored if not understood O.o

Reply:
yes, they are ignored, with a warning, but with Werror it result in a failure ;-)

Reply:
We should disable that warning then...